projects/gnutls/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM ossfuzz/base-libfuzzer
+MAINTAINER alex.gaynor@gmail.com
+RUN apt-get install -y make autoconf automake libtool autopoint libnettle6 nettle-dev pkg-config gperf bison autogen texinfo curl
+
+RUN git clone https://gitlab.com/gnutls/gnutls.git
+RUN cd gnutls && git submodule update --init
+
+# Using the client_corpus transcripts from boringssl, they're a decent starting point.
```

Comment:
I've been thinking if I need to put some variances in what the client or server supports. For instance the list of supported ciphers, curves, ... But for the server fuzzer it's up to the fuzzer to do variances in the client hello, and for the client fuzzer we'll could add or remove more support, but the fuzzer will end up selecting things.  So I currently don't think this will have much effect. I just need to add the support in both the client and server for as much as possible, which as you indicate might actually invalidate some of the existing corpus, but then find more.